src/test/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepTest.java

```diff
@@ -549,4 +553,16 @@ public void invalidChoiceParameterValue() throws Exception {
         j.assertLogContains("Value for choice parameter 'letter' is 'c', but valid choices are [a, b]",
                 j.assertBuildStatus(Result.FAILURE, us.scheduleBuild2(0)));
     }
+
+    @Test public void buildTriggerStep() throws Exception {
+        SnippetizerTester st = new SnippetizerTester(j);
+        BuildTriggerStep step = new BuildTriggerStep("downstream");
+        st.assertRoundTrip(step, "build 'downstream'");
+        step.setParameters(Arrays.asList(new StringParameterValue("branch", "default"), new BooleanParameterValue("correct", true)));
+        if (StringParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class)) {
+            st.assertRoundTrip(step, "build job: 'downstream', parameters: [string(name: 'branch', value: 'default'), booleanParam(name: 'correct', value: true)]");
```

Comment:
Ok, the reason this passes but the actual UI doesn't work is that `Snippetizer.assertRoundTrip` doesn't test that snippet generation roundtrips correctly, only that an already-configured Step roundtrips with the given text when converted to Groovy (i.e. DSL.invokeStep). You could delete config.jelly entirely, so that the snippet generator UI was totally broken for the step, and `SnippetizerTester.assertRoundTrip` wouldn't care. In our case, the problem is in configuring the step from the UI, but that is not tested.

Reply:
I think this is resolved by https://github.com/jenkinsci/pipeline-build-step-plugin/pull/27#issuecomment-511021550 right?

Reply:
Yes, closing.